modules/acquia_cms_headless/tests/src/Functional/HeadlessHybridModeTest.php

```diff
@@ -98,4 +139,13 @@ protected function installModule(string $module) {
     return $this->moduleInstaller->install([$module]);
   }
 
+  protected function childMenuTest($menu, $parent, $childs, $selector) {
```

Comment:
@rajeshreeputra Add return type for this method.

Reply:
+1

Reply:
done